Sources/Prototyper/Model/APIHandler.swift

```diff
@@ -99,11 +99,10 @@ private let defaultBoundary = "------VohpleBoundary4QuqLuM1cE5lMwCy"
 // MARK: - APIHandler
 ///This class deals with making all the HTTP calls to the prototyper api, to login, send feedback and share request.
 class APIHandler: ObservableObject {
-    var settings: PrototyperSettings
-    
-    /// The variable that is updated when the user logs in.
-    var isLoggedIn: Bool = false
-    
+    ///The URL for the prototyper instance
+    let prototyperInstance: URL
+    ///The prototyper state which is set durring the initialization
+    var state: PrototyperState
```

Comment:
I would suspect that the reference to the `PrototyperState` from the `APIHandler` causes a reference loop as the `PrototyperState` also has a reference to the `APIHandler`. I would suggest to move all Information that is relevant to the `APIHandler` to the `APIHandler` and if needed expose it as computed property in the `PrototyperState` to keep the implementation of the `APIHandler` not exposed to the rest of the Prototyper framework.